src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -73,6 +73,10 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!-- Report on low severity vulnerabilities, and higher. Allowed values are: low, moderate, high, critical -->
     <NuGetAuditLevel Condition=" '$(NuGetAuditLevel)' == '' ">low</NuGetAuditLevel>
     <!-- Report known vulnerabilities on direct dependencies only. Allowed values are: direct, all -->
+    <NuGetAuditMode Condition="'$(NuGetAuditMode)' == ''
+                    AND '$(NuGetExeSkipSdkAnalysisLevelCheck)' != 'true'
+                    AND $([MSBuild]::VersionGreaterThanOrEquals($([MSBuild]::ValueOrDefault('$(NETCoreSdkVersion)', '0.0')), '9.0.100'))"
```

Comment:
Are we supposed to use the `SdkAnalysisLevel` property instead of `NETCoreSdkVersion`?

Reply:
yes. If a customer is using analysis level 1 by explicitly setting it in their Directory.Build.props, and they don't want a new .NET SDK to add or change warnings, then we need to use their requested analysis level, not the SDK version. This is how it works with C# analysis levels, where you want your project targetnet `net8.0` to compile the same regardless if you're using the .NET 8 SDK or .NET 9 SDK.

Reply:
Thanks, Andy, for the explanation. Does this mean that we should be using `SdkAnalysisLevel` property instead of `NETCoreSdkVersion` property?

```suggestion
                    AND $([MSBuild]::VersionGreaterThanOrEquals($([MSBuild]::ValueOrDefault('$(SdkAnalysisLevel)', '0.0')), '9.0.100'))"
```

Reply:
Oops 🤦

I didn't notice when I was reviving this PR, or from your first comment. Sorry!   Good catch, I've just pushed an update that fixes this oversight.